docs/security/access-control.md

```diff
@@ -0,0 +1,659 @@
+---
+title: "Access Control"
+slug: /security/access-control
+keyword: security
+license: "This software is licensed under the Apache License version 2."
+---
+
+## Overview
+
+Gravitino adopts RBAC and DAC. 
+
+Role-based Access Control (RBAC): Access privileges are assigned to roles, which are in turn assigned to users or groups.
+
+Discretionary Access Control(DAC): Each metadata object has an owner, who can in turn grant access to that object.
+
+:::info
+
+Gravitino only supports authorization and doesn't support metadata authentication.
```

Comment:
Could you explain this more clearly?

Reply:
I don't know to how to make it clearer.

Reply:
What is `metadata authentication` ? and As I know Gravitino doesn't support authorization.

Reply:
`authorization` means that write privileges to underlying system.
`medata authentication` means that check privileges about metadata, if has privileges to allow the operation.

Reply:
I think you should add this to the document to clarify it.

Reply:
Added.